From jkosina@suse.cz  Thu Sep  2 13:04:41 2010
Date: Thu, 2 Sep 2010 16:34:36 +0200 (CEST)
From: Jiri Kosina <jkosina@suse.cz>
To: Greg KH <greg@kroah.com>
Cc: Stephen Rothwell <sfr@canb.auug.org.au>,
	linux-next@vger.kernel.org, linux-kernel@vger.kernel.org,
	JiebingLi <jiebing.li@intel.com>, Joe Perches <joe@perches.com>
Subject: USB: langwell: Remove unnecessary return's from void functions
Message-ID: <alpine.LNX.2.00.1009021633020.26813@pobox.suse.cz>

From: Joe Perches <joe@perches.com>

There are about 2500 void functions in drivers/usb
Only a few used return; at end of function.

Standardize them a bit.

Signed-off-by: Joe Perches <joe@perches.com>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Cc: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/langwell_udc.c |    5 -----
 1 file changed, 5 deletions(-)

--- a/drivers/usb/gadget/langwell_udc.c
+++ b/drivers/usb/gadget/langwell_udc.c
@@ -254,7 +254,6 @@ static void ep0_reset(struct langwell_ud
 	}
 
 	dev_vdbg(&dev->pdev->dev, "<--- %s()\n", __func__);
-	return;
 }
 
 
@@ -1502,7 +1501,6 @@ static void langwell_udc_start(struct la
 	writel(usbcmd, &dev->op_regs->usbcmd);
 
 	dev_dbg(&dev->pdev->dev, "<--- %s()\n", __func__);
-	return;
 }
 
 
@@ -1525,7 +1523,6 @@ static void langwell_udc_stop(struct lan
 	writel(usbcmd, &dev->op_regs->usbcmd);
 
 	dev_dbg(&dev->pdev->dev, "<--- %s()\n", __func__);
-	return;
 }
 
 
@@ -2417,7 +2414,6 @@ delegate:
 	}
 end:
 	dev_vdbg(&dev->pdev->dev, "<--- %s()\n", __func__);
-	return;
 }
 
 
@@ -2633,7 +2629,6 @@ static void handle_trans_complete(struct
 	}
 done:
 	dev_vdbg(&dev->pdev->dev, "<--- %s()\n", __func__);
-	return;
 }
 
 
